xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -536,6 +536,11 @@ bool CPVRChannelGroups::DeleteGroup(const CPVRChannelGroup &group)
     CLog::Log(LOGERROR, "PVR - %s - cannot delete internal group '%s'", __FUNCTION__, group.GroupName().c_str());
     return false;
   }
+  if (group.GroupID() <= 0)
+  {
+    CLog::Log(LOGDEBUG, "PVR - %s - cannot delete group '%s' invalid id '%i'", __FUNCTION__, group.GroupName().c_str(), group.GroupID());
```

Comment:
it's Sunday and 28 degrees out here so I'm not behind a keyboard all day ;-)
I'd return true, as the result is the same as for the other `return true` instances here: there's no group in the db